_posts/2024-09-26-jump_dev_2025.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: page
+title:  "JuMP-dev 2025"
+date:   2024-09-26
+categories: [jump-dev]
+permalink: /meetings/jumpdev2025/
+---
+
+JuMP-dev 2025 will be held November 17--20, 2025 in Auckland, New Zealand.
+
+This is the week following [CPAIOR 2025](https://sites.google.com/view/cpaior2025),
+which will be held November 10--13, 2025 in Melbourne, Australia. (Melbourne is
+a 4 hour flight and 2 hour time difference from New Zealand.)
+
+## Outline
+
+The purpose of JuMP-dev is to bring together students, researchers, and
+practitioners with interests in the methodological, algorithmic, and software
+aspects of [JuMP](https://github.com/jump-dev/JuMP.jl) and related packages. In
+particular, we invite new contributors and those who have not met the core
+development team.
+
+All participants will uphold the [JuMP Code of Conduct](https://github.com/jump-dev/JuMP.jl/blob/master/CODE_OF_CONDUCT.md).
+
+## Schedule
+
+The workshop will be structured as two days of talks, a hackathon, and a local
+excursion. Schedule is subject to change at any time.
+
+ * Monday 17 November: talks at GridAKL
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon at GridAKL
+
+## Key dates
+
+ * September 27 2025: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
+ * February 28 2025: talk submissions close
+ * March 31 2025: talk accept notifications
+ * November 17--20 2025: JuMP-dev 2025
```

Comment:
Program committee: thoughts on these dates?

Reply:
I'm assuming talk submissions open should be September 27 2024 instead?